plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -117,10 +120,12 @@
      *            RegEx patterns of files to exclude from the report
      * @param includePattern
      *            RegEx patterns of files to include in the report
+     * @deprecated This constructor is called internally only, but if you need to use it (for some strange reason), call
+     *            {@link #FindBugsPublisher()} and available setters
      */
     // CHECKSTYLE:OFF
     @SuppressWarnings("PMD.ExcessiveParameterList")
-    @DataBoundConstructor
+    @Deprecated
```

Comment:
I was doing it as a rule, as we don't have real information about non-public plugins extending this one.
I would keep it, but you are the maintainer, so if it's fine for you I can delete it. Please, confirm.